variables.sh.template

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
```

Comment:
Where does this file get used? Is this for deployment?

Reply:
If I was to guess, this would be for dev testing locally